docs/reclaimspace.md

```diff
@@ -121,8 +121,31 @@ $ kubectl annotate namespace default "reclaimspace.csiaddons.openshift.io/schedu
 namespace/default annotated
 ```
 
-**Note** Please note that the PersistentVolumeClaim annotation takes priority
-over Namespace annotation. The kubernetes-csi-addons only generate a
-`ReclaimSpaceCronJob` if the annotation exists on the Namespace. If an admin
-needs to modify or delete the annotation on the Namespace, they must perform
-the same action on all the PersistentVolumeClaims within that Namespace.
+**Note** Please note that the PersistentVolumeClaim annotation takes priority over Namespace
+annotation. The kubernetes-csi-addons only generate a `ReclaimSpaceCronJob` if the annotation
+exists on the Namespace. If an admin needs to modify or delete the annotation on the Namespace,
+they must perform the same action on all the PersistentVolumeClaims within that Namespace.
+
+## Annotating StorageClass
+
+You can create `ReclaimSpaceCronJob` CR automatically by adding the
+`reclaimspace.csiaddons.openshift.io/schedule: "@midnight"` annotations to the StorageClass object.
+This will only affect new PersistentVolumeClaims created from this StorageClass for ReclaimSpace
+operations. To include existing PersistentVolumeClaims for ReclaimSpace operations, you must restart
```

Comment:
@iPraveenParihar there will be a follow-up PR to remove this behavior is that correct?

Reply:
Yes, there will be a follow up PR on that.